homeassistant/components/electric_kiwi/__init__.py

```diff
@@ -54,6 +58,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except ApiException as err:
         raise ConfigEntryNotReady from err
 
+    if entry.unique_id is not None and entry.unique_id.startswith(DOMAIN):
+        ek_session = await ek_api.get_active_session()
+        unique_id = "_".join(str(num) for num in ek_session.customer_numbers)
+        hass.config_entries.async_update_entry(entry, unique_id=unique_id)
```

Comment:
Let's do a minor version migration

Reply:
Also double check if you have any entities that need a migration of unique id

Reply:
entities are fine, I checked them, I set them up properly the first time :) , minor version migration, let me go look into that.

Reply:
done